lib/useQuery.js

```diff
@@ -79,19 +78,12 @@ export function useQuery(serviceName, options = {}, queryHookOptions = {}) {
 
   const handleRealtimeEvent = useCallback(
     payload => {
-      if (disposed.current) return
```

Comment:
`useRealtime` shouldn't be calling this when disposed, so this shouldn't be needed (and wouldn't work in strict mode)